server/config.ts

```diff
@@ -134,6 +134,7 @@ export default {
   },
   dashboards: {
     probationPractitioner: {
+      percentageOfPaginationUsers: Number(get('SP_PERCENTAGE_OF_DASHBOARD_PAGINATION_USERS', '0')),
```

Comment:
_nit_ It doesn't look like `dashboards.probationPractitioner.percentageOfPaginationUsers` is used anywhere (PP dashboards are always on) but it's not a blocker for this change

Reply:
Ok sure David, I will remove it.